Azure-Samples/chat-with-your-data-solution-accelerator
# Migrating to Poetry from requirements.txt

## Purpose

Required for #170 
With this change it will be easier to maintain dependabot updates coming across requirements.txt files earlier. It combines all packages in pyproject.toml.

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
- Get the code
- Build the dev container (all the dependencies should get installed)
- For Non dev container setup execute -> pip install .
- Run the application
